src/test/factories/cycle.js

```diff
@@ -1,9 +1,13 @@
 import faker from 'faker'
+import moment from 'moment'
 
 import {CYCLE_STATES} from 'src/common/models/cycle'
 import {Cycle} from 'src/server/services/dataService'
 
 const now = new Date()
+const mondayOfCycleWeek = moment(now).startOf('isoweek')
+const mmddyy = mondayOfCycleWeek.clone().format('MM-DD-YY')
+const unixTimestamp = mondayOfCycleWeek.clone().format('x')
```

Comment:
With formatting handled on the client, this can be made into a util function to be used by presentational components like `ProjectList` (recommended placement: `/src/common/util/format.js`).